tests/components/weather/test_init.py

```diff
@@ -155,7 +157,7 @@ async def test_temperature(
     """Test temperature."""
     hass.config.units = unit_system
     native_value = 38
-    state_value = convert_temperature(native_value, native_unit, state_unit)
```

Comment:
These changes are not related to Mazda and should be in a different PR.

Reply:
This PR was to cover every integration at once, since there are only two affected